.github/workflows/publish-release.yml

```
@@ -1,9 +1,11 @@
 name: Publish Releases
 
 on:
+  tags:
```

actionlint (1.7.4)

unknown Webhook event "tags". see https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)